.pylintrc

```diff
@@ -9,7 +9,9 @@ extension-pkg-whitelist=
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=
+#
+# Files under alembic are nonconform.
```

Comment:
nit: `Files under alembic are generated and are nonconforming.`

Reply:
done